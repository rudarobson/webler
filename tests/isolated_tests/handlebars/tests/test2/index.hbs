{{#extend "default" title="Title test 2"}}
	{{#content "styles"}}
		<link type="text/css" rel="sylesheet" href="index.css" />
	{{/content}}

	{{#content "scripts"}}
		<script type="text/javascript" src="index.js"></script>
	{{/content}}

	{{#content "body"}}
		<div>Test 2</div>
	{{/content}}

{{/extend}}
