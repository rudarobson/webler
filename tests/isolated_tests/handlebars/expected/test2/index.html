<!DOCTYPE html><html><body><head><style type="text/css">.a-default-class-in-layout {}</style><link type="text/css" rel="sylesheet" href="index.css"><title>Title test 2</title></head><div>Test 2</div><script type="text/javascript" src="index.js"></script></body></html>